src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Steps/EligibleApprenticeshipRequestedSteps.cs

```diff
@@ -76,8 +74,8 @@ private async Task SetupApplicationAndApprenticeship(string status)
         {
             using (var dbConnection = new SqlConnection(TestContext.SqlDatabase.DatabaseInfo.ConnectionString))
             {
-                await dbConnection.ExecuteAsync($"insert into IncentiveApplication(id, accountId, accountLegalEntityId, dateCreated, status, dateSubmitted, submittedBy) values " +
-                                                $"(@id, @accountId, @accountLegalEntityId, @dateCreated, '{status}', @dateSubmitted, @submittedBy)", _incentiveApplication);
+                await dbConnection.ExecuteAsync($"insert into IncentiveApplication(id, accountId, accountLegalEntityId, dateCreated, status, dateSubmitted, submittedByName) values " +
```

Comment:
Should this also include the submitted by email address so that it's indicative of what will be submitted for an application?

Reply:
i don't think these fields are being asserted on here but i'll add email for completeness.